_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,140 @@
+
+### Inventory Summary (TBD)
+
+### Scanner Summary (TBD)
+
+### Violation Notifications
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+The following options are available, on a per resource basis. You can mix and
+match any combination of notifiers for each resource.
+
+This example shows adding a email, slack, and gcs notifer on cloudsql violations.
+
+```
+notifier:
+    resources:
+        - resource: cloudsql_acl_violations
+          should_notify: true
+          pipelines:
+             - name: gcs_violations_pipeline
+               configuration:
+                 gcs_path: gs://path_to_foo_bucket
+             - name: email_violations_pipeline
+               configuration:
+                 sendgrid_api_key: foobar_key
+                 sender: forseti-notify@forsetisecurity.org
+                 recipient: foo@gmail.com,bar@gmail.com,baz@gmail.com
+             - name: slack_webhook_pipeline
+               configuration:
+                 webhook_url: https://hooks.slack.com/services/foobar
+```
+
+* `should_notify`: Controls whether violation for each resource should be sent.
+  `true` enables the notification, and `false` disables the notification.
+
+* `name`: The desired notifiers for each resource.  You can specify multiple
+  notifiers for each resource.
+
+  The name must match the actual module name for each notifier in 
+  `forseti-security/google/cloud/forseti/notifier/notifiers`,
+  e.g. `email_violations.py`
+
+  These notifiers can be:
+  1. `email_violations`
+  This emails all the violation data via SendGrid.  Multiple email recipients are
+  delimited by comma.
+
+  1. `slack_webhook`
+  This sends individual violations to a Slack channel via a Slack webhook.
+  Configure a webhook in your organization's Slack settings and set the `webhook_url`.
+
+  1. `gcs_violations`
+  This uploads all the violation data to a GCS bucket.
+
+* `data_format`: Either `csv` (default) or `json`.
+  Slack only support json type.
+
+### CSCC Findings
+
+Forseti violations can be outputted for integration with
+[Cloud Security Command Center](https://cloud.google.com/security-command-center) on GCP.
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+
+1. Navigate to the `notifier` > `violation` > `cscc` section.
+
+```
+notifier:
+    violation:
+      cscc:
+        enabled: true
+        gcs_path: gs://inventoryscanner-henry.appspot.com
+```
+
+* `enabled`: Controls whether CSCC findings should be sent.
+  `true` enables the notification, and `false` disables the notification.
+* `gcs_path`: The GCS bucket to upload Forseti violations as CSCC findings.
+  [Sign-up for the Cloud SCC alpha program here!](outputted for integration with Cloud Security Command Center on GCP. Sign-up for the Cloud SCC alpha program here!)
+
+
+## Usage
+
+Forseti Notifier can be invoked in 2 ways, as part of the scheduled cron job's
+workflow, or manually via the Forseti CLI.
```

Comment:
You can invoke Forseti Notifier as part of the scheduled cron job's workflow or manually using the Forseti CLI.

Reply:
Done, thanks.